src/Sentry.Unity/UnityLogException.cs

```diff
@@ -149,6 +164,215 @@ public SentryException ToSentryException()
             };
         }
 
+        private SentryException ConvertFullStackTrace()
+        {
+            if (LogStackTrace.StartsWith("0x")
+                && LogStackTrace.Contains("(Unity) StackWalker::GetCurrentCallstack"))
+            {
+                return ConvertWindowsFullStackTraceImpl();
+            }
+
+            return ConvertFullStackTraceImpl();
+        }
+
+        private static readonly Regex WindowsStackFramePattern = new Regex(
+            @"(?<addr>0x[0-9a-fA-F]+)\s*" + // 0x12345678
+            @"\((?<module>.+?)\)\s*" + // (Unity), (Mono JIT Code), ...
+            @"(\((?<wrapper>.+?)\)\s*)?" + // (wrapper managed-to-native),
+            @"(\[(?<file>.+?):(?<line>\d+)\]\s*)?" + // [File.cs:123]
+            @"(?<function>.+)"); // StackWalker::GetCurrentCallstack, UnityEngine.Logger:Log (UnityEngine.LogType,UnityEngine.Object,string,object[]),
+
+        private SentryException ConvertWindowsFullStackTraceImpl()
+        {
+            var frames = new List<SentryStackFrame>();
+            var lines = LogStackTrace.Split('\n');
+            foreach (var traceLine in lines)
+            {
+                var item = traceLine.TrimEnd('\r');
+                if (string.IsNullOrWhiteSpace(item))
+                {
+                    continue;
+                }
+
+                var match = WindowsStackFramePattern.Match(item);
+                if (!match.Success)
+                {
+                    // we did something wrong, failed the check
+                    Debug.Log("failed parsing " + item);

```

Comment:
We avoid calling into `Debug.Log` here because the SDK has its own control for logging. The users are allowed to turn it on or off and control the level. If we're left with Debug.Log in the other implementation that this was based from, we should probably change there too